app/src/main/java/de/s3xy/architecturesample/github/GithubAuthHelper.java

```diff
@@ -23,7 +21,7 @@ public static String buildAuthUrl() {
                 .addPathSegment("login")
                 .addPathSegment("oauth")
                 .addPathSegment("authorize")
-                .addQueryParameter("client_id", CLIENT_ID)
+                .addQueryParameter("client_id", GithubConfig.CLIENT_ID)
```

Comment:
These should be provided by Dagger.
It would help in testing to easily swap the m out and for other situations like when we need to change this from Static values to dynamic values.